### ISSUE TITLE
How to find test error ( with test data) using keras? In the code, only training and validation set are used. 

### ISSUE THREAD

**Issue by ramboforever** [fchollet/keras#7268]
How can I find test error? 
I followed the code from below link. I have test data also. 

@fchollet  How did you find test error in the below code?  Did I need to create separate folder for same?

https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html

**Comment by Dref360** [comment]
You can call evaluate on your test set. Documentation : https://keras.io/models/model/

**Comment by ramboforever** [comment]
Thanks for quick reply and help.  I already have model.h5 file. Now i have test folder with label. How exactly i can code to find test error? Please have look on the link.. He used validation set in keras model. How to proceed exactly ? @Dref360

**Comment by alekospj** [comment]
eisai protos naoume!
